src/components/PositionedOverlay/PositionedOverlay.tsx

```diff
@@ -153,6 +158,8 @@ export default class PositionedOverlay extends React.PureComponent<
   private handleMeasurement = () => {
     const {lockPosition, top} = this.state;
 
+    this.observer.disconnect();
```

Comment:
Disconnect before we perform our handle measurements as it would cause an infinite loop